src/representation_theory/character.lean

```diff
@@ -34,7 +36,7 @@ variables [monoid G]
 
 /-- The character of a representation `V : fdRep k G` is the function associating to `g : G` the
 trace of the linear map `V.ρ g`.-/
-def character (V : fdRep k G) (g : G) := linear_map.trace k V (V.ρ g)
+@[simp] def character (V : fdRep k G) (g : G) := linear_map.trace k V (V.ρ g)
```

Comment:
```suggestion
def character (V : fdRep k G) (g : G) := linear_map.trace k V (V.ρ g)
```
I'm not so sure about the simp here. Once the API for `character` develops a bit, it's not so clear that we will want simp to always unfold it.

Reply:
Fair enough, I removed it